modules/machine-health-checks-resource.adoc

```diff
@@ -10,55 +10,40 @@ The MachineHealthCheck resource resembles the following YAML file:
 .MachineHealthCheck
 [source,yaml]
 ----
-apiVersion: healthchecking.openshift.io/v1alpha1
+apiVersion: machine.openshift.io/v1beta1
 kind: MachineHealthCheck
 metadata:
- name: example <1>
- namespace: openshift-machine-api
-Spec:
-  Selector:
+  name: example <1>
+  namespace: openshift-machine-api
+spec:
+  selector:
     matchLabels:
-      machine.openshift.io/cluster-api-cluster: <cluster_name> <2>
-      machine.openshift.io/cluster-api-machine-role: <label> <3>
-      machine.openshift.io/cluster-api-machine-type: <label> <3>
-      machine.openshift.io/cluster-api-machineset: <cluster_name>-<label>-<AWS-zone> <4>
+      machine.openshift.io/cluster-api-cluster: <infrastructureID> <2>
+      machine.openshift.io/cluster-api-machine-role: <role> <3>
+      machine.openshift.io/cluster-api-machine-type: <role> <3>
+      machine.openshift.io/cluster-api-machineset: <cluster_name>-<label>-<zone> <4>
+  unhealthyConditions:
+  - type:    "Ready"
+    status:  "Unknown"
+    timeout: "300s"
+  - type:    "Ready"
+    status:  "False"
+    timeout: "300s"
+  maxUnhealthy: "40%"
+status:
```

Comment:
The `status` field in this example can be removed. Status is automatically updated to the MachineHealthCheck after creation, it's not for user to specify.